client/my-sites/domains/domain-management/dataviews/domains-dataviews.tsx

```diff
@@ -140,3 +146,7 @@ export const DomainsDataViews = ( {
 		</>
 	);
 };
+
+export const DomainsDataViews = connect( null, { successNotice, errorNotice } )(
```

Comment:
All changes in this file reflect the attempt to switch to the hook instead of using the component.